keps/2724-topology-aware-scheduling/README.md

```diff
@@ -743,6 +766,91 @@ to ungate a Pod. The expectation is fulfilled if the Pod is observed as ungated
 or the ungating request fails. We hold ungating if there are pending ungatings
 within the PodSet.
 
+### Support for ProvisioningRequests
+
+We are going to support autoscaling via ProvisioningRequest AdmissionCheck.
+
+The key assumptions of the solution:
+- When there is ProvisioningRequest on the list of AdmissionChecks Kueue
+  Scheduler only reserves quota, but skips TAS assignment until all
+  AdmissionChecks (including the ProvisioningRequest) are `Ready` (green).
```

Comment:
Should the admission check return a signal to enable the 2nd pass TAS, rather than hardcoding this for PR?

Reply:
I think for now the ProvisioningRequest would be the only consumer of such an API. Indeed, in the future other requests like "KarpenterProvisioningRequest" may have a similar requirement, but I think we can then make it as a new feature / improvement. If we have more examples of such ACs it would also help us guide API naming. At the moment I'm not sure what would be the valid abstraction. 

Would it be "RequiresDelayedTopology", or "RequiresSecondPassScheduling", or "MayModifyNodes".
So, I would avoid committal API decisions and defer it for later when we have use cases, as a new feature of supporting other ACs.